source/fuzz/fact_manager.cpp

```diff
@@ -1565,6 +1573,10 @@ bool FactManager::IdIsIrrelevant(uint32_t result_id) const {
   return irrelevant_value_facts_->IdIsIrrelevant(result_id);
 }
 
+std::unordered_set<uint32_t> FactManager::GetIrrelevantIds() const {
```

Comment:
Same here - use a reference.

Reply:
Also, maybe add some very trivial tests for this method to `fact_manager_test.cpp`

Reply:
Done